templates/master/00-master/baremetal/files/baremetal-fletcher8.yaml

```diff
@@ -0,0 +1,16 @@
+filesystem: "root"
+mode: 0755
+path: "/usr/local/bin/fletcher8"
+contents:
+  inline: |
+    #!/usr/libexec/platform-python
+    import sys
+    
+    data = map(ord, sys.argv[1])
+    ckA = ckB = 0
+    
+    for b in data:
+        ckA = (ckA + b) & 0xf
+        ckB = (ckB + ckA) & 0xf
+    print((ckB << 4) | ckA )
```

Comment:
This file could use some sort of comment with what it's doing.  I don't find `fletcher8` very self-explanatory :wink:

Reply:
TBH, neither do I. :)

This commit is just moving a bunch of stuff from openshift-metal3/dev-scripts. It could use some cleanup.

Reply:
It generates an 8-bit id for the keepalived routers. I'll add a comment stating such.